### DISCUSSION TITLE
Is it possible to get the pointer position with the `onWheel` gesture?

### DISCUSSION THREAD

**Discussion by baptisteArno** [pmndrs/use-gesture#557] [Q&A] [answered]
It seems that the `xy` or `initial` props in the `onWheel` handler always return [0, 0].

I need the point position to be able to "zoom" in where the pointer points. Am I missing something?

**Comment by dbismut** [comment, marked as answer]
I would strongly advise to use onPinch for anything related to zooming. Also there's a sandbox in the demo folder called card-zoom that might help.

**Reply by baptisteArno** [reply]
Oh, so `onPinch` is triggered from mouse wheel event when using a mouse? I can't test, I don't have a mouse 🤪

**Reply by dbismut** [reply]
When pressing the control key yes, as this is what's the convention. But you can also trigger onPinch without any key by setting the modifierKey option to null https://use-gesture.netlify.app/docs/options/#modifierkey

**Reply by Todilo** [reply]
Is this still the case? I set the modifier on the pinch to null and now I get pinch event on mousewheel. 
I get an origin but no delta (0,0) and offset is some very strange value
